### PR TITLE
[WIP] TPC: Added expert visulization task for creating ND monitoring data

### PR DESCRIPTION
**Pull request by bulukutlu**
Created a new task that generates THnSparse files to be used in the TPC by the detector experts

### CODE REVIEW

**Comment by bulukutlu** [comment]
This depends on https://github.com/AliceO2Group/AliceO2/pull/8354

File: Modules/TPC/include/TPC/ExpertVis.h [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+{
+
+/// \brief TPC ExpertVis QC Task
+/// It is final because there is no reason to derive from it. Just remove it if needed.
+/// \author Berkin Ulukutlu & Jens Wiechula
+class ExpertVis final : public TaskInterface
+{
+ public:
+  /// \brief Constructor
+  ExpertVis();
```
Comment (wiechula) [inline Modules/TPC/include/TPC/ExpertVis.h:38]:
```suggestion
  ExpertVis() = default;
```

File: Modules/TPC/src/ExpertVis.cxx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#include "TPC/ExpertVis.h"
+
+namespace o2::quality_control_modules::tpc
+{
+
+ExpertVis::ExpertVis() : TaskInterface() {}
+
+ExpertVis::~ExpertVis()
+{
+}
```
Comment (wiechula) [inline Modules/TPC/src/ExpertVis.cxx:36]:
```suggestion
```

File: Modules/TPC/src/ExpertVis.cxx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+void ExpertVis::initialize(o2::framework::InitContext& /*ctx*/)
+{
+  ILOG(Info, Support) << "initialize TPC ExpertVis QC task" << ENDM;
+
+  mQCExpertVis.initializeHistograms();
+  getObjectsManager()->startPublishing(&mQCExpertVis.getHistogramPID());
+  getObjectsManager()->startPublishing(&mQCExpertVis.getHistogramTracks());
```
Comment (wiechula) [inline Modules/TPC/src/ExpertVis.cxx:44]:
```suggestion
  getObjectsManager()->startPublishing(mQCExpertVis.getHistogramPID());
  getObjectsManager()->startPublishing(mQCExpertVis.getHistogramTracks());
```
After changes in O2

**Review by wiechula — CHANGES_REQUESTED** [review]

**Review by wiechula — APPROVED** [review]

**Comment by stheckel** [comment]
As just discussed on the corresponding O2 PR, further tests and potentially reconsideration is needed, so please also mark this one here again as draft, so the CI does not continuously try to rebuild.

**Comment by Barthelemy** [comment]
Hello, 
This PR has been in progress for quite some time. Would you mind if we closed it and you reopen when it is ready ? 
Cheers,
Barth

**Comment by bulukutlu** [comment]
Further tests needed investigating memory and space requirements, will open again when ready.

**Pull request closed without merging**